### DISCUSSION TITLE
Converting emoji shorcode into image when converting from markdown to docx

### DISCUSSION THREAD

**Discussion by zero0n3** [jgm/pandoc#10365] [General]
Hello
when converting from markdown to doc, is it possible to convert the emoji shortcode, present in a markdown file, into the actual emoji in the docx?

Thanks

**Comment by jgm** [comment]
You need to enable the emoji extension:
```
pandoc -f markdown+emoji in.md -o out.docx
```

**Reply by zero0n3** [reply]
Thanks @jgm . I tried (pandoc 3.5 on win). What could be the problem why, using that command on a very simple md file, I get a tofu box?

This is the md text:

```
**hello world**

:smile:
```

This is the result:

![immagine](https://github.com/user-attachments/assets/5f336cd0-a5c7-4c79-bc42-be04f99ed15f)

**Reply by alerque** [reply]
The font stack in your office app needs to support a font (or at least a fallback) that has the relevant glyph. This is not converting the input to an image, it is converting it to the most relevant Unicode code point. Drawing that code point as a color emote glyph is up to the font/shaper/renderer stack you're using.

**Reply by zero0n3** [reply]
Thanks @alerque ! That was exactly the problem. Sorry if I hadn't thought of that 😄 

> The font stack in your office app needs to support a font (or at least a fallback) that has the relevant glyph. This is not converting the input to an image, it is converting it to the most relevant Unicode code point. Drawing that code point as a color emote glyph is up to the font/shaper/renderer stack you're using.
